### DISCUSSION TITLE
Poll: Next Feature

### DISCUSSION THREAD

**Discussion by rumboalla** [rumboalla/apkupdater#425] [Polls]
Now that 3.0.0 is here, what feature would you like to see next?

**Comment by benliberti** [comment]
APKPure source.

**Reply by rumboalla** [reply]
Updated the poll

**Reply by benliberti** [reply]
Cheers!

**Comment by AnonymousWP** [comment]
Voted. However, I'm not a programmer so could you explain the essential differences between SharedPreferences and DataStore?

**Reply by rumboalla** [reply]
They both do the same (store key-value pairs on disk), but DataStore is more modern and will work better with the current tech stack (compose, flow). For the user nothing will change, but it should simplify the code in many parts of the app and bring some small performance increases.

**Reply by AnonymousWP** [reply]
That's nice! Because I'm committed to this project and really love it to see you picked it up again (honestly, after 2.x I had low hopes for any release in the future, but I'm glad I was wrong), I'll continue to test betas (or even alphas) if they will release.

I don't know how much work it is to migrate, but will 1 feature be implemented in this list for the next build?

**Reply by rumboalla** [reply]
Yes, that's the idea. although not necessarily for next version since some of these features are quite big and will take a while.

**Comment by vipulm7** [comment]
Google play support is must!
